catchup/catchpointService.go

```diff
@@ -339,6 +339,8 @@ func (cs *CatchpointCatchupService) processStageLastestBlockDownload() (err erro
 	if ledgerBlock, err := cs.ledger.Block(blockRound); err == nil {
 		blk = &ledgerBlock
 	}
+	var protoParams config.ConsensusParams
+	var ok bool
 
```

Comment:
nit: the variables can be declared inside the for loop for tighter scoping, since they are not needed outside.

Reply:
My intention here was to avoid having these variables get re-initialized on each iteration. ( it won't cause any issue if they would, though )